test-parser.js

```diff
@@ -0,0 +1,111 @@
+// Copyright 2012 Dan Wolff | danwolff.se
```

Comment:
Why 2012 and who is copyright owner?

Reply:
I split the original file in two, preserving history. Though github implies this is new content, it is in fact unchanged since the original file was created.

The copyright headers in both files could be revised, though I feel it is a separate cleanup task unrelated to this PR's scope.